### PR TITLE
Parallax and Properties.GetColor

### PR DESCRIPTION
**Pull request by tmathews**
This PR adds parallax properties for layers as well as adds a helper method for custom properties to parse color information.

### CODE REVIEW

**Comment by lafriks** [comment]
Can you add `parallaxx` and `parallaxy` also to to layer, group and objectgroup types?

**Comment by tmathews** [comment]
Sure thing.

**Comment by tmathews** [comment]
Updated

File: tmx_property.go [hunk 1]
```diff
@@ -103,3 +107,24 @@ func (p Properties) GetFloat(name string) float64 {
 	}
 	return 0
 }
+
+func (p Properties) GetColor(name string) color.Color {
```
Comment (lafriks) [inline tmx_property.go:111]: please do not add mutliple unrelated changes/features into single PR

Comment (tmathews) [reply tmx_property.go:111]: No problem.

**Comment by tmathews** [comment]
See: https://github.com/lafriks/go-tiled/pull/80
